apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/SourceApiConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi;
+
+import io.stargate.sgv2.api.common.grpc.SourceApi;
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.Produces;
+
+@ApplicationScoped
+public class SourceApiConfiguration {
+
+  @Produces
+  @SourceApi
+  public String sourceApi() {
+    return "rest";
```

Comment:
Looks like "rest" is used here because `AuthApi` has no enum for docs-api.
Might want to add a comment mentioning this since I first thought this was unintentional.

Reply:
See comment above..